clickhouse_client_settings.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" ?>
+<config>
+    <max_partitions_per_insert_block>10000</max_partitions_per_insert_block>
```

Comment:
@forus you shouldn't change this conf field

Reply:
@mkarmona Yes, as I could overcome the issue by changing the sql declarations as you've noted [here](https://github.com/opentargets/genetics/issues/337#issuecomment-496540199)?

Reply:
I use the master branch that does run into this limitation with recent CH. We should remove this setting in `19.05.05` when the partitions have been changed.